src/sentry/integrations/source_code_management/metrics.py

```diff
@@ -35,6 +35,22 @@ class SourceCodeIssueIntegrationInteractionType(Enum):
     SYNC_STATUS_OUTBOUND = "SYNC_STATUS_OUTBOUND"
     SYNC_ASSIGNEE_OUTBOUND = "SYNC_ASSIGNEE_OUTBOUND"
 
+    def __str__(self) -> str:
+        return self.value.lower()
+
+
+class SourceCodeSearchEndpointInteractionType(Enum):
```

Comment:
i think the merge conflict will force u to move all of this into a common SCMInteraction type